velox/common/caching/SsdFile.cpp

```diff
@@ -66,28 +69,6 @@ void disableCow(int32_t fd) {
 #endif // linux
 }
 
-// TODO: Remove this function once we migrate all files to velox fs.
```

Comment:
We still have checkpoint file write though posix api? thanks!

Reply:
Yes